build.gradle

```diff
@@ -1,11 +1,11 @@
 buildscript {
     repositories {
         jcenter()
-        google()
+        maven { url 'https://maven.google.com' }
```

Comment:
Because when I use `google()` my build fails and I didn't feel like figuring out why :-)